src/Workspaces/CSharp/Portable/Extensions/StatementSyntaxExtensions.cs

```diff
@@ -13,6 +13,26 @@ namespace Microsoft.CodeAnalysis.CSharp.Extensions
 {
     internal static class StatementSyntaxExtensions
     {
+        public static bool IsFirstStatementInEnclosingBlock(this StatementSyntax statement)
+        {
+            if (!(statement.Parent is BlockSyntax enclosingBlock))
+            {
+                return false;
+            }
+
+            return statement == enclosingBlock.Statements.First();
```

Comment:
can be simplified to just `=> statement.Parent is BlockSyntax bloc && block.Statements.First() == statement;`